Scfmodule5/src/MathOperation.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * This class used to find least common multiple and highest 
+ * common factor of two number.
+ */
+public class MathOperation {
+
+	/*
+	 * This method is used to find least common multiple of two number.
+	 * @param firstNumber This is first number.
+	 * @param secondNumber This is second number.
+	 * @return This return least common multiple of first number and second number.
+	 */
+   public int lcm(int firstNumber, int secondNumber)
+   {
+	
+	   return lcmCalc(firstNumber, secondNumber, 1);
+    }
+	
+	
+	/*
+	 * This method is used to find least common multiple of two number.
+	 * @param firstNumber This is first number.
+	 * @param secondNumber This is second number.
+	 * @param commonMultiple This is common multiple of both number.
+	 * @return This return least common multiple of first number and second number.
+	 */
+	public int lcmCalc(int firstNumber, int secondNumber,int commonMultiple) {
```

Comment:
If this method is not called outside from this class, can we declare this as private method.